tasmota/tasmota_xdrv_driver/xdrv_27_esp32_shutter.ino

```diff
@@ -687,7 +687,7 @@ void ShutterInit(void)
           Shutter[i].min_realPositionChange = 0;
         break;
         case SHT_COUNTER:
-          Shutter[i].min_realPositionChange = SHT_DIV_ROUND(Shutter[i].min_realPositionChange, Shutter[i].motordelay);
+          Shutter[i].min_realPositionChange = SHT_DIV_ROUND(Shutter[i].min_realPositionChange, Shutter[i].motordelay > 0?Shutter[i].motordelay : 1);
```

Comment:
Is there no better way to make sure ``Shutter[i].motordelay`` well never become 0? User entry checks?

Reply:
Good point. This is a special requirement only for shuttermode 4 and the stepper motors. On all other shuttermodes the correct and default value is 0. If checked on userinput I would need to change it when you change the shuttermode and additionally check on the cmdShutterMotordelay what is the current shuttermode and reject. For sure possible, but maybe not less code

Reply:
OK. Let's do it this way then if see if it at least solves the exception 6. Thx.

Reply:
Something I often use when I want to avoid a zero value is an OR with 1
```
result = value | 1;
```
Of course it may modified the value but depending on use case, it might not be a problem
And it's very light on code